html/images/os/lenovo.svg

```diff
@@ -1 +1,34 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 256 38.573" version="1"><path d="M237.947 25.444c1.198-3.875 1.198-6.16.05-7.65-.855-1.026-2.103-1.539-3.995-1.539-4.217 0-6.553 2.395-8.435 8.394-1.208 3.936-1.208 6.22-.06 7.7.795.976 2.114 1.49 3.996 1.49 4.157 0 6.562-2.345 8.444-8.395zm16.487-8.676c1.65 2.064 2.003 4.68 1.026 7.881-2.737 8.847-13.064 13.92-28.363 13.92-9.129 0-15.339-1.771-18.087-5.193-1.65-2.054-1.932-4.73-.966-7.932 2.738-8.847 13.065-13.92 28.363-13.92 9.069 0 15.35 1.822 18.027 5.244m-57.059-4.388h16.205c-.976 1.308-18.6 24.991-18.892 25.394h-24.87c-.122-.745-3.09-24.478-3.202-25.394h15.239c.11.745 2.224 17.805 2.224 17.805s13.065-17.463 13.296-17.805M147.1 25.444c1.198-3.875 1.258-6.16.06-7.65-.804-1.026-2.113-1.539-3.935-1.539-4.227 0-6.622 2.395-8.444 8.394-1.258 3.936-1.258 6.22-.06 7.7.795.976 2.113 1.49 3.935 1.49 4.227 0 6.623-2.345 8.444-8.395zm16.547-8.676c1.661 2.064 2.003 4.68.977 7.881-2.738 8.847-13.075 13.92-28.303 13.92-9.13 0-15.35-1.771-18.097-5.193-1.65-2.054-1.993-4.73-1.027-7.932 2.798-8.847 13.125-13.92 28.363-13.92 9.13 0 15.35 1.822 18.087 5.244m-47.698-1.248c.624.796.966 1.59.795 2.738l-6.33 19.567h-16.78l5.94-18.258c.22-.795-.061-1.48-.514-1.993-.805-.976-2.516-1.6-8.394-1.198 0 0-6.955 21.167-7.076 21.449H66.822c.16-.453 7.87-24.015 7.981-24.247 5.828-1.027 13.528-1.993 21.52-1.993 10.84 0 17.452 1.309 19.626 3.936m-61.226 5.364c.343-1.6-.17-2.798-.744-3.482-.624-.805-2.003-1.832-4.852-1.832-4.046 0-7.075 1.942-8.102 5.193l-1.993 7.479 15.691-7.358zm14.091-.06c0 1.198-.281 2.224-.513 2.677-.513.12-29.782 6.623-29.782 6.623s-.292 1.43.795 2.174c1.43 1.027 2.97 1.308 5.586 1.42 6.452.281 15.188-3.483 17.634-4.56-.393 1.248-2.909 6.733-3.02 7.015-1.087.403-7.136 2.335-19.002 2.395-4.51.06-12.551-.513-15.863-4.68-1.65-2.053-1.942-4.962-.805-8.615 2.687-8.727 12.732-13.75 27.447-13.75 8.223 0 13.588 1.54 16.044 4.56.966 1.258 1.48 2.859 1.48 4.74M11.696 0h16.607c-.342 1.077-11.525 37.14-11.706 37.774H0C.342 36.687 11.524.624 11.696 0" fill="#005e9d"/></svg>
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please can you compress this svg down. You can use this to do that: https://jakearchibald.github.io/svgomg/